### PR TITLE
Bug 1599526 - Push Health handle tasks with timout errors 

### PR DESCRIPTION
**Pull request by camd**

### CODE REVIEW

File: treeherder/push_health/tests.py [hunk 1]
```diff
@@ -89,12 +89,10 @@ def get_current_test_failures(push, option_map):
     # for the same job (with different sub-tests), but it's only supported by
     # postgres.  Just using .distinct() has no effect.
     new_failure_lines = FailureLine.objects.filter(
-        action='test_result',
+        action__in=['test_result', 'log'],
         job_log__job__push=push,
         job_log__job__result='testfailed',
         job_log__job__tier__lte=2
-    ).exclude(
```
Comment (camd) [inline treeherder/push_health/tests.py]: ``test`` will be null for a ``log`` line.

File: tests/autoclassify/utils.py [hunk 2]
```diff
@@ -39,6 +40,8 @@ def create_failure_lines(job, failure_line_list,
         data.update(base_data)
         data.update(updates)
         failure_line = FailureLine(**data)
+        job_log = JobLog.objects.create(job=job, name='foo', url='bar{}'.format(i), status=1)
```
Comment (camd) [inline tests/autoclassify/utils.py]: Needed to add a ``log_line`` to connect it to a ``job`` because that's how we do it in ``tests.py``

**Comment by camd** [comment]
I have to fix a couple of old tests.  But it'll just be tweaks.  The code for this feature won't change.

File: tests/autoclassify/utils.py (hunk 2 above)
Comment (armenzg) [reply tests/autoclassify/utils.py]: Do we need to fix `tests.py` instead?
I'm confused with the use of `foo` and `bar`.

Comment (camd) [reply tests/autoclassify/utils.py]: In this case, `foo` is just the name given to the log and `bar#` would be the URL.  But we won't actually be using those values.  They have to be unique, due to an index, but other than that, they're just a link to the job in question.

**Pull request merged**